types/three/src/helpers/ArrowHelper.d.ts

```diff
@@ -25,9 +41,12 @@ export class ArrowHelper extends Object3D {
     );
 
     /**
-     * @default 'ArrowHelper'
+     * A Read-only _string_ to check if `this` object type.
+     * @remarks Sub-classes will update this value.
+     * @override
+     * @defaultValue `ArrowHelper`
      */
-    type: string;
+    override readonly type: string | 'ArrowHelper';
```

Comment:
I'm curious why this can't be typed as just `'ArrowHelper'` here and in the rest of the files.

Reply:
This is because of the base `Object3D` type. And the need for `type` de overridden in many sub-classes. You can only override a string type, not a string const. 
At the end Typescript will only consider this be a string, ignoring the `ArrowHelper` string const. But I decided put it there just for documentation and to help the developers when looking at the type definition to see what the default value he can expect.